src/server/utils/logging/config/otel.js

```diff
@@ -53,16 +43,24 @@ export function createOtelTransport() {
     };
   }, {}) : {};
 
-  let logRecordProcessorOptions = {
-    recordProcessorType: 'batch',
-    exporterOptions: { protocol: 'grpc' },
-  };
+  let logRecordProcessorOptions = [];
+
+  if (grpcExporter) {
+    logRecordProcessorOptions.push({
+      recordProcessorType: 'batch',
+      exporterOptions: { protocol: 'grpc' },
+    });
+  }
 
-  if (process.env.NODE_ENV === 'development') {
-    logRecordProcessorOptions = [logRecordProcessorOptions, {
+  if (consoleExporter) {
+    logRecordProcessorOptions.push({
       recordProcessorType: 'simple',
       exporterOptions: { protocol: 'console' },
-    }];
+    });
+  }
+
+  if (logRecordProcessorOptions.length === 1) {
+    [logRecordProcessorOptions] = logRecordProcessorOptions;
```

Comment:
is this intended for `logRecordProcessorOptions` to either be empty array or array of objects or single object`{}` ?

Reply:
Yes